ext/acorn/acorn-rx/src/test/java/com/nhaarman/acorn/presentation/RxSceneTest.kt

```diff
@@ -51,6 +51,18 @@ class RxSceneTest {
         expect(observer.lastValue).toBe(Option.just(testView))
     }
 
+    @Test
+    fun `subscribing after view attach`() {
+        /* Given */
+        scene.attach(testView)
+
+        /* When */
+        val observer = scene.viewObservable.test()
+
+        /* Then */
+        expect(observer.lastValue).toBe(Option.just(testView))
```

Comment:
wil je niet kijken naar alle values?

Reply:
This is the only value emitted for this Observer